UE/Application/ISmsDb.hpp

```diff
@@ -15,7 +15,7 @@ class ISmsDb
     virtual Sms* getOne(int id) = 0;
     virtual void addOne(Sms sms) = 0;
     virtual void remove() = 0;
-
+    virtual Sms* getLast() = 0;
```

Comment:
I would add a mental note that maybe we should be passing references instead of pointers. But maybe not in this story, just when we refactor the whole project

Reply:
Sure, It's worth to remember it while we will be refactoring. Although I can still change It now if others think the same as you.